.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!-- 
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka-HTTP: https://github.com/akka/akka-http/issues 
+ - Alpakka:   https://github.com/akka/alpakka/issues 
+ - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - ...
+
```

Comment:
Let's add one sentence about that questions are not welcome here.

Reply:
Ok adding